app/models/datatable.rb

```diff
@@ -74,7 +74,7 @@ def per_page
   def sort_order
     colnum = 0
     sort_by = []
-    while true
+    loop do
```

Comment:
Is this recommended by rubocop?
Anyway, I don't like ``while true`` nor ``loop do`` ...